src/test/munit/socket-custom-protocol-config.xml

```diff
@@ -11,7 +11,7 @@
 
     <import file="socket-base-config.xml"/>
 
-    <munit:config name="Custom tcp protocol"/>
+    <munit:config name="Custom tcp protocol" ignore="true"/>
```

Comment:
The ignore property should be placed in the test tag with an expression.

Reply:
If I place the ignore on the test only, the build fails